pkg/ast/ast_value.go

```diff
@@ -5,6 +5,9 @@ import (
 	"fmt"
 	"io"
 
+	"github.com/tidwall/sjson"
```

Comment:
`jsonparser` can do it too 😛

Reply:
I'm using jsonparser extensively. Sometimes sjson is a bit more convenient though.